smokey/src/main/java/frc/robot/commands/AllignWithTag.java

```diff
@@ -0,0 +1,83 @@
+// ************************************************************
+// Bishop Blanchet Robotics
+// Home of the Cybears
+// FRC - Charged Up - 2023
+// File: AllignWithTag.java
+// Intent: Forms a command to allign itself with a designated april tag.
+// ************************************************************
+
+// ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ 
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.math.controller.PIDController;
+import frc.robot.subsystems.DrivetrainSubsystem;
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import frc.robot.common.MotorUtils;
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableInstance;
+
+public class AllignWithTag extends CommandBase{
+    private boolean done = false;
+    private double tagID;
+    private double velocityValue = 0.3;
+    private PIDController yPID = new PIDController(1.0, 0.0, 0.0);
+    private DrivetrainSubsystem drivetrainsubsystem = null;
+    NetworkTable table = NetworkTableInstance.getDefault().getTable("limelight");
+    
+    public AllignWithTag(double tagID, DrivetrainSubsystem drivetrainSubsystem){
+        this.tagID = tagID;
+
+        this.drivetrainsubsystem = drivetrainSubsystem;
+
+        addRequirements(drivetrainsubsystem);
+    }
+
+    @Override
+    public void initialize() {
+        done = false;
+        drivetrainsubsystem.drive(new ChassisSpeeds(0,0,0));
+    }
+
+    @Override
+    public void execute() {
+        double tid = table.getEntry("tid").getDouble(0.0);
```

Comment:
What is 'tid'?  In Microsoft Entra / Azure Active Directory 'tid' is a claim in an OAuth token short for tenant identifier.  I presume that is not what this 'tid' means.  Idea here is to convey something to the next person that reads this about what the variable means.

Reply:
A question about this line is what does .getDouble(0.0) do?

Reply:
.getDouble gets the entry as a double. the value 0.0 is the default value to return.

Reply:
added a comment for what 'tid' means (tag ID) and changed variable names to also make it more clear.